assembl/static2/js/app/client.js

```diff
@@ -28,6 +31,9 @@ const myFragmentMatcher = new IntrospectionFragmentMatcher({
 // http://dev.apollodata.com/react/query-splitting.html
 
 const dataIdFromObject = o => o.id;
+const useCSRFProtection = document.getElementById('useCSRFProtection')
```

Comment:
It's a bit confusing here for two reasons:
- the returned value is a string
- the values are 'true' and 'false' formatted as string

Shouldn't we type useCSRFProtection as boolean and map 'true' to TRUE and 'false' to FALSE so that the condition below can be `if (useCSRFProtection)` instead of `if (useCSRFProtection === 'true')`

What do you think ?

Reply:
I would normally but the config comes from the HTML field and I thought it 'safer' to work using strings rather than assuming booleans would be preserved in the <... value=true>